src/pykeen/datasets/literal_base.py

```diff
@@ -46,12 +48,12 @@ def __init__(
             self._load_validation()
 
     def _load(self) -> None:
-        self._training = TriplesNumericLiteralsFactory(
+        self._training = TriplesNumericLiteralsFactory.from_path(
```

Comment:
should now be `self.triples_factory_cls.from_path`?

Reply:
e889ea1